nautobot/docs/plugins/development.md

```diff
@@ -579,7 +579,7 @@ With this code, once your plugin is installed, the Git repository creation/editi
 
 ### Populating Extensibility Features
 
-In many cases, a plugin may wish to make use of Nautobot's various extensibility features, such as [custom fields](../../additional-features/custom-fields) or [relationships](../../models/extras/relationship/). It can be useful for a plugin to automatically create a custom field definition or relationship definition as a consequence of being installed and activated, so that everyday usage of the plugin can rely upon these definitions to be present.
+In many cases, a plugin may wish to make use of Nautobot's various extensibility features, such as [custom fields](../../models/extras/customfield/) or [relationships](../../models/extras/relationship/). It can be useful for a plugin to automatically create a custom field definition or relationship definition as a consequence of being installed and activated, so that everyday usage of the plugin can rely upon these definitions to be present.
```

Comment:
Just for the sake of consistency!

```suggestion
In many cases, a plugin may wish to make use of Nautobot's various extensibility features, such as [custom fields](../../models/extras/customfield.md) or [relationships](../../models/extras/relationship.md). It can be useful for a plugin to automatically create a custom field definition or relationship definition as a consequence of being installed and activated, so that everyday usage of the plugin can rely upon these definitions to be present.
```